src/foremast/s3/s3apps.py

```diff
@@ -67,7 +71,11 @@ def create_bucket(self):
                 LOG.error("Shared bucket %s does not exist", self.bucket)
                 raise S3SharedBucketNotFound
         else:
-            _response = self.s3client.create_bucket(ACL=self.s3props['bucket_acl'], Bucket=self.bucket)
+            if self.region == 'us-east-1':
```

Comment:
Could this leverage the variable default_region? Would be helpful for other users who maybe have us-west-2 as their default for example.

Reply:
oh duh, just missed that. Didnt want us-east-1 hardcoded anywhere

Reply:
Oh, actually us-east-1 is hardcoded on purpose. It is the only value where it needs to be true, even if the default region is something else, you need to include the `CreateBucketConfiguration` part of the command.